Lib/defconQt/fontInfo.py

```diff
@@ -131,9 +131,13 @@ def loadPositiveIntegerFloat(self, font, src, dst):
     def loadIntegerFloatList(self, font, src, dst):
         values = " ".join(str(val) for val in getattr(font.info, src))
         setattr(self, dst + "Edit", QLineEdit(values, self))
+        validator = QRegularExpressionValidator(self)
+        validator.setRegularExpression(
+            QRegularExpression("(-?\d+(.\d+)?\s*)*"))
+        getattr(self, dst + "Edit").setValidator(validator)
 
     def writeIntegerFloatList(self, font, src, dst):
-        values = getattr(self, src + "Edit").text()
+        values = getattr(self, src + "Edit").text().split()
```

Comment:
cc @moyogo: the lack of split() wasn't caught by the tests because such things are written independently in the tests folder. Any way to test writing too?